resources/lang/en/authorization.php

```diff
@@ -12,6 +12,8 @@
     'unauthorized' => 'Access denied.',
 
     'beatmap_discussion' => [
+        'discussion_locked' => 'This beatmap is locked for discussion.',
```

Comment:
not sure if worth moving the key esp when it's still called 'beatmap discussion' (instead of beatmapset) - also if anything this key is probably more applicable to the original (only discussion specifically locked) instead of the current one (beatmapset also "locked")...